src/disk/d64.rs

```diff
@@ -7,19 +7,21 @@ use crate::disk::bam::{BAMFormat, BAMSection};
 use crate::disk::block::{BlockDevice, BlockDeviceRef, ImageBlockDevice};
 use crate::disk::error::DiskError;
 use crate::disk::header::{Header, HeaderFormat};
-use crate::disk::{self, BAMRef, Disk, DiskFormat, Geometry, Id, Image, Location, Track, BAM};
+use crate::disk::{self, BAMRef, Disk, DiskFormat, Geometry, Image, Location, Track, BAM};
 
 /// A description of the header format for this disk image type.
 static HEADER_FORMAT: HeaderFormat = HeaderFormat {
     location: Location(18, 0),
     first_directory_offset: 0x00,
-    disk_dos_version_offset: 0x02,
+    diskette_format_type_offset: 0x02,
```

Comment:
Should this be `disk_` instead of `diskette_` to follow existing naming?

Reply:
I was on the fence about this. I was torn between being consistent with the exact terminology of the CBM DOS source ("diskette"), or being brief and consistent with the other fields ("disk"). I've changed this to `disk_`.